taxi/admin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from taxi.models import Manufacturer, Car, Driver
```

Comment:
Add a blank line between different groups of imports according to PEP-8. Add this change to all imports that are affected by this issue.